website_product_configurator/tests/test_website_product_configurator_values.py

```diff
@@ -2,14 +2,12 @@
 
 from odoo import fields
 
-from odoo.addons.product_configurator.tests import test_product_configurator_test_cases
+from odoo.addons.product_configurator.tests import common
 
 
-class TestProductConfiguratorValues(
-    test_product_configurator_test_cases.ProductConfiguratorTestCases
-):
+class TestProductConfiguratorValues(common.ProductConfiguratorTestCases):
     def setUp(self):
-        super(TestProductConfiguratorValues, self).setUp()
+        super().setUp()
```

Comment:
We should change this to `setUpClass` instead.
Looks like the base class is doing it already: https://github.com/OCA/product-configurator/blob/e78af0c423193a6e466f22567377fdcb0e1b9e46/product_configurator/tests/common.py#L5C1-L7C29